Makefile

```diff
@@ -1,8 +1,14 @@
+UUID = transparent-top-bar@zhanghai.me
+
 .PHONY: build
 build: $(wildcard src/*)
 	mkdir -p build/
 	gnome-extensions pack -fo build src
 
+.PHONY: install
+install:
```

Comment:
shouldn't this target depend on the build output? but I don't want to hard-code this either. I'll try to resolve this myself.

Reply:
I felt it probably should, but wasn't sure of your programming style. Glad to see you've sorted this :)